landavailability/api/serializers.py

```diff
@@ -1,8 +1,38 @@
 from .models import BusStop
 from rest_framework import serializers
+from django.contrib.gis.geos import GEOSGeometry
+import json
 
 
 class BusStopSerializer(serializers.ModelSerializer):
+    srid = serializers.IntegerField(write_only=True)
```

Comment:
I guess this will make an appearance in a few different APIs, but it won't be immediately obvious why it's there? Short comment?